cloud-sql/postgres/knex/index.js

```diff
@@ -311,7 +240,7 @@ app.get('/', async (req, res) => {
   }
 });
 
-app.post('/', async (req, res) => {
+const httpPost = app.post('*', async (req, res) => {
```

Comment:
`httpPost` looks like it points to an Express route, not the route handler function itself.

We probably need something like this:
```
const httpPost = async (req, res) => {
   ...
}

app.post('*', httpPost)
```